k8s/aat/money-claims/claim-store/claim-store.yaml

```diff
@@ -43,5 +51,6 @@ spec:
       subscriptionId: "1c4f0704-a29e-403d-b719-b90c34ef14c9"
       tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
     tags:
-      postgresql-pod: false
-      cmc-citizen-frontend: false
+      cmc-claim-store-postgresql-pod: false
```

Comment:
do you need specific tags for each app?
the benefit of using the generic tag is that it applies everywhere to postgres pod behaviour